templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Driver</h1>
+
+  <p>Are you sure you want to delete the driver: {{ driver }}</p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger"/>
```

Comment:
It's recommended to add a name attribute to the submit button for better accessibility. For example: `<input type="submit" name="delete-driver" value="Yes, delete" class="btn btn-danger"/>`.